company/company.de.robomarkup

```diff
@@ -0,0 +1,93 @@
+// Pseudo markup format for Robo Instructus containing the company messages received through the game's story.
+// Spoiler alert!
+//
+// `#!company` means the start of a message, the lines following are the message in simplified markdown.
+//  Most of the time the first line, starting with '# ' is the subject.
+
+
+// intro text, should be exactly 3 lines
+#!company id{c-await}
+Bergungsingenieur,
+Ankunftsbericht bestätigt
+Anweisungen abwarten.
+
+
+#!company id{c-receiving}
+Kommunikation empfangen
+
+
+#!company id{c-ack}
+Bestätigt
+
+
+#!company id{c-arrive}
+# Re: Iceni Bergungsankunftsbericht
+
+Du hast also einen Roboter gefunden? Sieht süß aus, wenn auch etwas primitiv. Aber wenn das die beste Bergung ist, die Sie finden können ... Nun, sagen wir einfach, sie deckt nicht einmal die Kosten der wertvollen Nährstoffsuppenrationen, die Sie zweifellos genießen, während Sie dies lesen.
```

Comment:
In English _"cost of the value nutri-soup rations"_ the _"value"_ means "cheap". It comes from supermarkets lowest cost ranges.

Reply:
It was meant in a more ironical way here, so I retranslated it here to valuable. But now that I look at it, I think we should change it